docs/release-notes/cloud/cloud-2023-07.md

```diff
@@ -0,0 +1,90 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud July 2023.
+included_jira_versions: ['2023.07']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.5.46
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.6.39
+---
+
+# Cloud July 2023
+
+These release notes are for the Codacy Cloud updates during July 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added support for AI-generated context-aware comments and fix suggestions directly in your Git provider. For more details, [see the announcement on our blog](https://blog.codacy.com/codacy-quality-ai-actionable-fixes/). (CY-6855, HRZ-286)
```

Comment:
Should we note that this needs to be enabled in settings?

Reply:
the linked announcement already includes the instructions to get started ☝️

Reply:
We usually keep these notes short and just notify what's available, so we avoid adding instructions or caveats here.